lib/Twig/Loader/Chain.php

```diff
@@ -166,4 +166,12 @@ public function isFresh($name, $time)
 
         throw new Twig_Error_Loader(sprintf('Template "%s" is not defined%s.', $name, $exceptions ? ' ('.implode(', ', $exceptions).')' : ''));
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTemplateFreshness(Twig_CacheInterface $cache)
+    {
+        return time();
```

Comment:
this means that as soon as several loaders are used, any logic implemented in the Filesystem loader gets ignored and you will get ``time()`` each time. I fail to see how this API can be useful